### ISSUE TITLE
Add the ability to hide the address bar button.

### ISSUE THREAD

**Issue by dougwl** [josh-berry/tab-stash#153]
#### > The settings should include this option as it is covered by the API, even though is not a serious issue it wouldn't hurt being able to choose if the button is something the user want.

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Yup I agree, I think it's unfortunate that Firefox chose to remove this option; I don't understand what they were thinking here. I'll see what I can do about adding an option in a future release.

Glad to hear you're enjoying Tab Stash!

**Comment by JaxxArmstrong** [comment]
I fully agree. While this isn't an operational issue, it's still an eyesore when you're applying a minimalist approach to it. I'm looking forward to a toggle in the addon options.

Thank you for your blood, sweat and tears creating and maintaining this (Firefox) addon.
